applications/registration-react/package.json

```diff
@@ -6,8 +6,9 @@
   "scripts": {
     "start": "webpack-dev-server --hot --inline --config webpack.hostdev.config.js",
     "build": "webpack --config webpack.prod.config.js -p",
-    "lint:all": "(cd ../../; npx eslint applications/registration-react --ext=.js,.jsx --fix)",
-    "lint": "(cd ../../; npx lint-staged)",
```

Comment:
Det er en grunn at disse kommandoene var skrevet med referanse til FDK. @markkopaas har en bra forklaring. Men for å nevne noe, så er det at git repo er FDK, og ikke registration-react. Dermed er git hooks installert 2 nivåer høyere.